Documentation/guides/tracing.md

```diff
@@ -98,6 +98,63 @@ And the output files should be found in the current directory. You can
 use the `-o` switch if you would prefer to output them to a specific
 directory.
 
+## How to get GC memory dumps?
+
+If running on desktop, you can simply do:
+
+```
+$ dotnet gcdump collect -p 29096
```

Comment:
This should use `--process-id` instead of `-p` -- I had no idea what `-p 29096` was supposed to mean, which is why I went looking for `dotnet gcdump --help` -- and the surrounding context should explicitly mention that the Process ID must be provided to `dotnet gcdump collect --process-id $PID`.

A "larger example" would be helpful too:

```
# `hw-readline` is ye standard Hello World, with a `Console.ReadLine()` at the end
% dotnet run --project hw-readline.csproj
Hello, World!
Press <ENTER> to continue

# Then from another shell…

# Determine which process ID to dump
% ps | grep 'dotnet.*hw-re'
33972 ttys049    0:01.86 dotnet run --project hw-readline.csproj
33977 ttys050    0:00.00 grep dotnet.*hw-re

# Collect the GC info
% dotnet gcdump collect --process-id 33972
Writing gcdump to '…/hw-readline/20230314_113922_33972.gcdump'...
	Finished writing 5624131 bytes.
```

Also of note is that the process being dumped *cannot* be backgrounded; it *must* be running.  (i.e. hitting `^Z` and trying to use the same shell to both launch the process *and* dump memory is Doomed To Fail™.)